doc/Uplink.xml

```diff
@@ -210,19 +271,22 @@
-      <para>The service shall authenticate itself at the remote client using TLS client authentication according to RFC 5246 or subsequent specifications.</para>
-      <para>To uniquely identify local service on remote client, it is recommended to have a unique client certificate installed on each local service. For example, CN field or Serial number of installed certificate could be used to uniquely identify the local service.</para>
-      <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is
-        configured for the configuration without the need of further authentication request headers. </para>
-    </section>
-    <section>
-      <title>HTTP/2 Frames</title>
-      <para>Once an http/2 connection has been established the communication parameters are negotiated as specified by RFC 7540 so that the uplink can be used to exchange frames between the remote client and the local service.</para>
-    </section>
-    <section>
-      <title>HTTP Transactions</title>
-      <para>The uplink shall be used in reverse direction for http requests and responses. The remote client shall send requests that are served in a standard http manner by the local service.</para>
+      <para>The remote client shall authenticate itself using a valid server certificate. The service shall verify the validity of the remote certificate according to RFC 6125.</para>
+      <section>
+        <title>mTLS authentication</title>
+        <para>When using mTLS authentication, the service shall authenticate itself at the remote client using TLS client authentication according to RFC 5246 or subsequent specifications.</para>
+        <para>To uniquely identify local service on remote client, it is recommended to have a unique client certificate installed on each local service. For example, CN field or Serial number of installed certificate could be used to uniquely identify the local service.</para>
+        <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is configured for the configuration without the need of further authentication request headers. </para>
+        <para>If the <literal>CertificateID</literal> is present in the <literal>UplinkConfiguration</literal> the device shall use this authentication mechanism.</para>
+      </section>
+      <section>
+        <title>JWT authentication</title>
+        <para>When using JWT authentication, the local service shall provide a JWT token when sending the upgrade request.</para>
+        <para>The remote client shall validate the JWT token provided following the JWT specification as defined in the Core specification.</para>
+        <para>The device shall assign any tunneled HTTP or RTSP request the UserLevel that is configured for the configuration without the need of further authentication request headers.</para>
```

Comment:
Update "tunneled HTTP or RTSP request the UserLevel" to "tunneled HTTP or RTSP request at the UserLevel"

Reply:
```suggestion
        <para>The device shall assign any tunnelled HTTP or RTSP request at the UserLevel that is configured for the configuration without the need of further authentication request headers.</para>
```